config/playground/eks/dev/providers.tf

```diff
@@ -0,0 +1,15 @@
+terraform {
```

Comment:
could add comment to head of file to explain what does this file do?

Reply:
Yes. Adds context in the head of file